_opensearch/rest-api/cluster-stats.md

```diff
@@ -217,7 +217,7 @@ Parameter | Type | Description
             "coordinating_only": 0,
             "data": 1,
             "ingest": 1,
-            "master": 1,
+            "cluster_manager": 1,
```

Comment:
Sorry for not describing this part in detail:
To keep the backwards compatibility and the logic simple, I preserved the `master` field here.
When the node has got either `master` role or `cluster_manager` role, the count of both `master` and `cluster_manager` will be increased by 1. So here it should be:
```
            "master": 1,
            "cluster_manager": 1,
```
Code: https://github.com/opensearch-project/OpenSearch/blob/2.0/server/src/main/java/org/opensearch/action/admin/cluster/stats/ClusterStatsNodes.java#L228